README.md

```diff
@@ -1 +1,40 @@
-# LeaderBoard-project
\ No newline at end of file
+# LeaderBoard-project
+
+
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+> This is the simple to do list: using webpack that shows of my accomplishments as a software developer.In this project  we learn how to setup the webpack and nodemodules. This project do the to-do-list add , remove and edit function with local storage.
+
+
+
+## Built With
+
+- HTML
+- Css
+- Javascript
+- Es6
+- webpack
+- gitflow
+
+## Getting Started
+
+To get a local copy up and running follow these simple example steps.
+
```

Comment:
- [ ] Your README file does not look professional enough, kindly give instructions on how your app can be used on the user local machine. This project is different from your previous projects, there is need for some dependencies to be installed. Kindly make the journey of your app users easier for them by giving them adequate information about how to run the app on their local machines.

![your-readme](https://user-images.githubusercontent.com/26597432/202615138-28c49148-cb7c-4e86-9389-4537e260bfb3.png)